tests/terraform/tfsec/insecure.tf

```diff
@@ -0,0 +1,6 @@
+resource "aws_security_group_rule" "example" {
```

Comment:
This tf file failed validate so it didn't return the tfsec error that is expected. Added the provider and other missing required fields for the resource and it functioned as expected.

```
docker run -v ./tests/terraform/tfsec:/iac seiso/easy_infra /bin/bash -c "terraform --skip-terrascan --skip-checkov init"
INFO:  Passed terraform init initialization
ERROR:  Failed terraform validate validation

Error: Missing required argument

The argument "region" is required, but was not set.
```

Reply:
Now working as expected; thanks for the catch!